recipes/newrelic/apm/dotNet/windows/net-install.yml

```diff
@@ -20,12 +20,18 @@ keywords:
 processMatch:
   - w3wp.exe
 
-validationNrql: "SELECT count(*) from Transaction WHERE host like '{{.HOSTNAME}}%' facet entityGuid since 10 minutes ago"
+validationNrql: "SELECT count(*) from Transaction WHERE host like '{{ .HOSTNAME }}%' facet entityGuid since 10 minutes ago"
```

Comment:
@jaffinito I dont think there should be spaces between the {{ and .VARNAME and .VARNAME }}

Reply:
Odd, I have always used spaces between the braces (for readability) and not seen an issue.